### ISSUE TITLE
Bug UI: Wider Screen 720p+ Touch/No Touch != Responsive?

### ISSUE THREAD

**Issue by raditzlawliet** [openivity/openivity.github.io#58]
I notice something unusual, In chrome we can test the layout is using different view (Touch/No Touch) with different treat (responsive)

Touch (Mobile/Desktop) - Large Screen
![image](https://github.com/openivity/openivity.github.io/assets/3763178/fc1ffab9-189a-4357-9aa8-58ddd37b25a0)

No Touch (Mobile/Desktop) - Large Screen
![image](https://github.com/openivity/openivity.github.io/assets/3763178/f8e1d686-4832-4786-88a4-515ed0d3c99b)

Do we need to make it different with Touch? I don't think so.
I think for now it's better to stick with Screen instead (Responsive)

**Comment by raditzlawliet** [comment]
This issue noticed after #54

**Comment by raditzlawliet** [comment]
I see, there's mixed style for touch
![image](https://github.com/openivity/openivity.github.io/assets/3763178/b2f509bb-26bc-4ceb-b605-bd594a3e6eba)
![image](https://github.com/openivity/openivity.github.io/assets/3763178/d4c26752-09fc-40fd-9b8e-a7eef1906a6c)

this style intended for touch screen, but because it's not restricted with screen-size (responsive), the wider screen still affected)

**Comment by raditzlawliet** [comment]
@muktihari

**Comment by muktihari** [comment]
Yes, layout for other than smartphone and destkop is currently have not been layouting properly. We can work on this later on the final re-layouting after all feature completed. Our current priority is how can we make component as independent as possible so when we do re-layouting it doesn't get affected (or affected at minimal)

**Comment by muktihari** [comment]
> Do we need to make it different with Touch? I don't think so.
>I think for now it's better to stick with Screen instead (Responsive)

Yes I agree, I did mention the last responsiveness after bootstrap migration doesn't not fit on my iphone mini, and this re-layouting on OpenActivity.vue which is just for trial and error, so it doesn't affect the legacy TheActivity.vue

**Comment by raditzlawliet** [comment]
That's good let use responsive for styling (pointer/touch will use as spesific case only).
Let keep this issue open and discuss again after re-layout plan come up later
